servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -97,17 +110,18 @@ def _to_top_labels_aggregations(label_summaries):
     # where valid means a label has jobs more than _LABEL_MIN_COUNT_FOR_RANK.
     label_freq = {}
     for label, item in label_summaries.items():
+        # job-id label contains one job per label value, filter it out
```

Comment:
This is not strictly true––one dsub job can have many tasks each of which is represented as a distinct job here. I'm fine with filtering it as an aggregation key though, would just update the comment: "Do not use the job-id label as an aggregation key".

Also, do you also want to filter out the 'task-id' label here?